testing/ios/IosUnitTests/README.md

```diff
@@ -1,16 +1,39 @@
 # iOS Unit Tests
 
-These are the unit tests for iOS engine.  They can be executed locally and are
+These are the unit tests for iOS engine. They can be executed locally and are
 also run in LUCI builds.
 
-## Running Tests
+## Running Tests from command line
 
-To build and run the iOS tests, run the following script from the src directory:
+To build and run the iOS tests, use the `run_tests` script from the `src` directory.
 
 ```sh
 flutter/testing/run_tests.py --type=objc
 ```
 
+And if you're on Apple Silicon:
+
+```sh
+./flutter/testing/run_tests.py \
+  --type=objc \
+  --variant=host_debug_unopt_arm64 \
+  --ios=ios_debug_sim_unopt_arm64
+```
+
+The `.xcresult` is automatically removed after testing ends. To change this:
+
+```sh
+export FLUTTER_TEST_OUTPUTS_DIR=~/Desktop
```

Comment:
We set `FLUTTER_TEST_OUTPUTS_DIR` env in our CI system, however if it's useful to be used on the command line `parser.add_argument` output option around here
https://github.com/flutter/engine/blob/9b716e031fd2160213534add72489f18ca4b8361/testing/run_tests.py#L1219
And use that and fallback to `FLUTTER_TEST_OUTPUTS_DIR` is currently used, first  if it's not passed in.

https://github.com/flutter/engine/blob/9b716e031fd2160213534add72489f18ca4b8361/testing/run_tests.py#L831
https://github.com/flutter/engine/blob/9b716e031fd2160213534add72489f18ca4b8361/testing/run_tests.py#L297

However if you don't want to touch the python code I understand. 🙂

Reply:
For now I'd prefer to leave it as it is (I'm low on time), but I might come back to it in the future and add a proper argument for that.